plutus-core/testlib/PlutusCore/Generators/Internal/Entity.hs

```diff
@@ -111,28 +107,16 @@ revealUnique (Name name uniq) =
 -- TODO: we can generate more types here.
 -- | Generate a 'Builtin' and supply its typed version to a continuation.
 withTypedBuiltinGen
-    :: (uni ~ DefaultUni, HasConstantIn uni term, Monad m)
-    => (forall a. AsKnownType term a -> GenT m c) -> GenT m c
-withTypedBuiltinGen k = Gen.choice
-    [ k @Integer       AsKnownType
-    , k @BS.ByteString AsKnownType
-    , k @Bool          AsKnownType
-    ]
-
--- | Generate a 'Term' along with the value it computes to,
--- having a generator of terms of built-in types.
-withCheckedTermGen
-    :: (uni ~ DefaultUni, fun ~ DefaultFun, Monad m)
-    => TypedBuiltinGenT (Plain Term uni fun) m
-    -> (forall a. AsKnownType (Plain Term uni fun) a ->
-            TermOf (Plain Term uni fun) (EvaluationResult (Plain Term uni fun)) ->
-                GenT m c)
```

Comment:
We no longer use this apparently. I'm curious what it was used for, but not too much for me to actually go and check.